src/java/org/apache/cassandra/db/compaction/unified/RealEnvironment.java

```diff
@@ -163,9 +163,9 @@ public double maxThroughput()
     @Override
     public String toString()
     {
-        if (metrics() == null)
-            return "Uninitialized environment";
-        return String.format("Read latency: %d us / partition, flush latency: %d us / KiB, compaction latency: %d us / KiB, bfpr: %f, measured WA: %.2f, flush size %s",
+        return String.format("Default Environment for %s - Read latency: %d us / partition, flush latency: %d us / KiB, " +
+                             "compaction latency: %d us / KiB, bfpr: %f, measured WA: %.2f, flush size %s",
+                             realm.metadata(),
```

Comment:
nit: since we assume not null realm we could make it explicit by adding assertion in the constructor